src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Views/BackgroundTask/Edit.cshtml

```diff
@@ -38,7 +38,7 @@
                 <div class="@Orchard.GetEndClasses()">
                     <div class="input-group mb-3">
                         <input asp-for="Schedule" class="form-control" />
-                        <button id="use-default-schedule" class="btn btn-outline-success">@T["Reset Schedule"]</button>
+                        <span id="use-default-schedule" class="btn btn-outline-danger">@T["Reset Schedule"]</span>
```

Comment:
Changed it to danger (red), because resetting something is more akin to deleting, which is usually red.